attributes/default.rb

```diff
@@ -0,0 +1,23 @@
+default['luigi']['setup_user'] = true
+default['luigi']['setup_group'] = true
+default['luigi']['user'] = 'luigi'
+default['luigi']['group'] = 'luigi'
+default['luigi']['config_dir'] = '/etc/luigi'
+
+default['luigi']['server']['auto_start'] = true
+default['luigi']['server']['log_dir'] = '/var/log/luigi'
+default['luigi']['server']['pid_file'] = '/var/run/luigid.pid'
+
+default['luigi']['server']['create_dir'] = true
```

Comment:
I think this parameter is not necessary. If `luigi.server.dir` does not exist, we always create it, that's it.